Code/GraphMol/SynthonSpaceSearch/SynthonSpace.h

```diff
@@ -109,6 +109,16 @@ struct RDKIT_SYNTHONSPACESEARCH_EXPORT SynthonSpaceSearchParams {
              // appropriate for Morgan fingerprints.  With RDKit fingerprints,
              // 0.05 is adequate, and higher than that has been seen to
              // produce long run times.
+  unsigned int minHitHeavyAtoms{0};  // Minimum number of heavy atoms in a hit.
+  unsigned int maxHitHeavyAtoms{0};  // Maximum number of heavy atoms in a hit.
+                                     // 0 means no maximum.
+  double minHitMolWt{0};             // Minimum molecular weight for a hit.
+  double maxHitMolWt{0};  // Maximum molecular weight for a hit.  0.0 means
+                          // no maximum.
+  unsigned int minHitChiralAtoms{
+      0};  // Minimum number of chiral atoms in a hit.
+  unsigned int maxHitChiralAtoms{0};  // Maximum number of chiral atoms in a
```

Comment:
Using an unsigned int for this means that you can't do searches where you explicitly don't want chiral centers (or potential chiral centers) in the molecule.

I'm not sure whether this is a concern or not

Reply:
I guess someone might want non-chiral hits.  I've changed the default to maximum to -1, and also maxHitHeavyAtoms for consistency.